src/fsharp/FSharp.Core/map.fsi

```diff
@@ -83,7 +83,16 @@ namespace Microsoft.FSharp.Collections
         /// <param name="value">A reference to the output value.</param>
         ///
         /// <returns><c>true</c> if the value is present, <c>false</c> if not.</returns>
-        member TryGetValue: key:'Key * [<System.Runtime.InteropServices.Out>] value:byref<'Value> -> bool
+        member TryGetValue: key: 'Key * [<System.Runtime.InteropServices.Out>] value: byref<'Value> -> bool
+        
+        /// <summary>The empty map.</summary>
+        static member Empty: Map<'Key,'Value> when 'Key: comparison
```

Comment:
I hadn't noticed this addition - `Empty` should be removed, as the function `Map.empty` is preferred, and we don't want both showing in autocomplete lists

Reply:
I remember now, I moved this from the additions in illib.fs. I'll refactor the illib.fs code to refer to Map.empty so the build is green again

Reply:
Thanks!